apps/api/src/feedback/feedback.e2e.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { Test, TestingModule } from '@nestjs/testing'
+import { AppModule } from '../app/app.module'
+import { FeedbackService } from '../feedback/service/feedback.service'
+import { MockMailService } from '../mail/services/mock.service'
+import {
+  FastifyAdapter,
+  NestFastifyApplication
+} from '@nestjs/platform-fastify'
+import { MAIL_SERVICE } from '../mail/services/interface.service'
+import { FeedbackModule } from './feedback.module'
+import { MailModule } from '../mail/mail.module'
+import { PrismaService } from '../prisma/prisma.service'
+import { User } from '@prisma/client'
+import cleanUp from '../common/cleanup'
+
+describe('Feedback Controller (E2E)', () => {
+  let app: NestFastifyApplication
+  let feedbackService: FeedbackService
+  let mockMailService: MockMailService
+  let prisma: PrismaService
+  let user: User
+
+  beforeAll(async () => {
+    const moduleRef: TestingModule = await Test.createTestingModule({
+      imports: [AppModule, FeedbackModule, MailModule]
+    })
+      .overrideProvider(MAIL_SERVICE)
+      .useClass(MockMailService)
+      .compile()
+
+    app = moduleRef.createNestApplication<NestFastifyApplication>(
+      new FastifyAdapter()
+    )
+    feedbackService = moduleRef.get(FeedbackService)
+    mockMailService = moduleRef.get(MAIL_SERVICE)
+
+    prisma = moduleRef.get(PrismaService)
+
+    await app.init()
+    await app.getHttpAdapter().getInstance().ready()
+
+    await cleanUp(prisma)
+
+    user = await prisma.user.create({
```

Comment:
Move user creation to `beforeEach`. Also, add a `afterEach` method in which you will delete the user.